design/one-pager-resource-reclaim-policy.md

```diff
@@ -0,0 +1,63 @@
+# Resource Reclaim Policy
+* Owner: Daniel Mangum (@hasheddan)
+* Reviewers: Crossplane Maintainers
+* Status: Draft
+
+## Terminology
+
+* **`External Resource`**: an infrastructure resource that runs outside of Crossplane (i.e. an S3 Bucket on AWS).
+* **`Concrete Resource`**: a Kubernetes resource that is responsible for managing an external resource (recieves configuration details from the `ResourceClass` and `ResourceClaim`). It is of type `Resource` but will be referred to as `Concrete Resource` consistently here to avoid confusion.
+* **`ResourceClass`**: a Kubernetes resource that contains implementation details specific to a certain environment or deployment, and policies related to a kind of resource.
+* **`ResourceClaim`**: a Kubernetes resource that captures the desired configuration of a resource from the perspective of a workload or application.
+
+## Background
+
+Crossplane resource classes allow for a `reclaimPolicy` to be set on creation. Acceptable values for `reclaimPolicy` are `Delete` or `Retain`. This value informs Crossplane of how to behave when a `Concrete Resource` is deleted. If the policy is set to `Delete`, the `External Resource` will be deleted when the `Concrete Resource` is deleted, which is generally triggered by the deletetion of a `Resource Claim` for that `Concrete Resource`. If set to `Retain`, the `Concrete Resource` will be deleted, but the `External Resource` will persist.
+
+*Note: `reclaimPolicy` is not a required field for a `ResourceClass`. If it is not supplied, the `Concrete Resource` will be created with it's default `reclaimPolicy`. Currenty, every `Concrete Resource` in Crossplane defaults to `Retain`.*
+
+## Workflow
```

Comment:
This is a good representation of how reclaim policies work in Crossplane today. What I'd like from this one pager is to help us decide whether reclaim policies _should_ work as they do in Crossplane today, or whether they should work more like the `PersistentVolume` reclaim policy that inspired them (or some third option?).